config/index.js

```diff
@@ -360,6 +360,38 @@ var conf = convict({
       doc: 'which key prefix to store sessions in redis under'
     }
   },
+  devicesCache: {
+    expiresIn: {
+      default: '1 hour',
+      format: 'duration',
+      env: 'CACHE_EXPIRES_IN',
+      doc: 'how long before devices cached expire'
+    },
+    generateTimeout: {
+      default: '2 seconds',
```

Comment:
So I guess this needs to be ~11 seconds now based on our profile server experience? 🕐